NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
```

Comment:
So here I took out the else statement but it seems that every time I call my  functions without the else statement I kept on getting undefined. Again I don't know how to solve this part of the issue.